jdaviz/cli.py

```diff
@@ -111,8 +116,8 @@ def _main(config=None):
                                      'loaded from FILENAME.')
     filepaths_nargs = '*'
     if config is None:
-        parser.add_argument('layout', choices=['cubeviz', 'specviz', 'specviz2d',
-                                               'mosviz', 'imviz'],
+        parser.add_argument('--layout', default='', choices=['cubeviz', 'specviz', 'specviz2d',
```

Comment:
I think this definitely deserves to be quite "loud" in the change log entry as it technically is a breaking change to the CLI.

Discussed offline: after the launcher is more finalized and ready for public use, we might want to update the docs to suggest `jdaviz filename` directly and only keeping `--layout` as advanced functionality.  But that can (and should) be deferred.